### PR TITLE
Release notes for 0.20.0

### PR DESCRIPTION
**Pull request by haadcode**
This PR will update CHANGELOG to contain description of the changes since 0.19.0. This PR is required to release 0.20.0.

***This is still work in progress!***

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+
+We've improved the documents by adding details, fixing errors and clarifying them.
+
+We also improved the tests a lot since the previous release. We now run the tests with js-ipfs-api (go-ipfs) in addition to running them with js-ipfs (Node.js). We've also cleaned up and refactored the boilerplate code for tests, improved the speed of the test run and generally polished the test suite for better readability and maintainability.
+
+### Custom stores and keys
+
+OrbitDB can now use a custom store through addDatabaseType(), see more [here]().
+
+We added support for pssing a custom [keystore]() to the store and use your own keys to sign and verify the database updates.
```
Comment (fazo96) [inline CHANGELOG.md]: sneaky typo here :eyes:

**Comment by Schwartz10** [comment]
Hey team! @shamb0t are there breaking changes coming in this release with respect to data structures / formats?

**Comment by shamb0t** [comment]
Hey @Schwartz10! :wave:  There will be breaking changes to the API but the new release will still be able to read old data structures and should be backwards compatible w.r.t. old OrbitDB addresses . The shape of new entries will change to include an `identity` and `cid` field and will have their version incremented to 1. 
Does that answer your question?

**Comment by fazo96** [comment]
What's blocking this?

**Comment by shamb0t** [comment]
Hey @fazo96 thanks for stopping by! No blockers but an issue was discovered and fixed in `orbit-db-keystore`, that meant a bit more work needed to be done to ensure old data-structures could still be read with the new keystore. Once all the keystore PRs https://github.com/orbitdb/orbit-db-keystore/pull/15, https://github.com/orbitdb/orbit-db-identity-provider/pull/23, https://github.com/orbitdb/ipfs-log/pull/238, are merged we should be ready for this and release. Almost there! :)

**Comment by haadcode** [comment]
@shamb0t @aphelionz I believe this still needs to be merged (and updated before?) for 0.20.0 release.

**Comment by haadcode** [comment]
(and rebase before merging)

**Comment by haadcode** [comment]
This looks good to me and ready to merge 👍

**Comment by haadcode** [comment]
Good catch @shamb0t! 👍 LGTM.

**Pull request merged**